<?xml version="1.0" encoding="UTF-8" ?>
<testsuites errors="1"
            failures="0"
            tests="1">

  <testsuite name="main"
    errors="1" failures="0" hostname="" id="" package="main" skipped="0"
    tests="1" time="0.02800" timestamp="Sat, 22 Oct 2016 08:03:03 GMT">
  
    <testcase name="抓取虎航的資料" time="0.02800" assertions="0">
    </testcase>
      <system-err>
          Error while running [Dist / Main / Main / 抓取虎航的資料]:

  ReferenceError: i is not defined
    at Object._ [as 抓取虎航的資料] (/Users/user/Sites/onlinereadbook/mini-hackathon/mini-project/01-chat-bot-ai/TeamA/tigerair/test-dist/main/main.js:48:16)
    at Module.call (/Users/user/Sites/onlinereadbook/mini-hackathon/mini-project/01-chat-bot-ai/TeamA/tigerair/node_modules/nightwatch/lib/runner/module.js:62:34)
    at /Users/user/Sites/onlinereadbook/mini-hackathon/mini-project/01-chat-bot-ai/TeamA/tigerair/node_modules/nightwatch/lib/runner/testcase.js:70:29
    at _fulfilled (/Users/user/Sites/onlinereadbook/mini-hackathon/mini-project/01-chat-bot-ai/TeamA/tigerair/node_modules/q/q.js:834:54)
    at self.promiseDispatch.done (/Users/user/Sites/onlinereadbook/mini-hackathon/mini-project/01-chat-bot-ai/TeamA/tigerair/node_modules/q/q.js:863:30)
    at Promise.promise.promiseDispatch (/Users/user/Sites/onlinereadbook/mini-hackathon/mini-project/01-chat-bot-ai/TeamA/tigerair/node_modules/q/q.js:796:13)
    at /Users/user/Sites/onlinereadbook/mini-hackathon/mini-project/01-chat-bot-ai/TeamA/tigerair/node_modules/q/q.js:556:49
    at runSingle (/Users/user/Sites/onlinereadbook/mini-hackathon/mini-project/01-chat-bot-ai/TeamA/tigerair/node_modules/q/q.js:137:13)
    at flush (/Users/user/Sites/onlinereadbook/mini-hackathon/mini-project/01-chat-bot-ai/TeamA/tigerair/node_modules/q/q.js:125:13)
    at _combinedTickCallback (internal/process/next_tick.js:67:7)
      </system-err>
  
  
  </testsuite>
</testsuites>
